### ISSUE TITLE
Cursor freezes in normal mode when pressing <c-h/j/k/l>

### ISSUE THREAD

**Issue by ghry5** [vscode-neovim/vscode-neovim#1133]
Anytime i hit one of the below keys the cursor completly freezes. I can still enter insert mode with `a`,`i`,... type and exit it but in normal mode it stays frozen. Keymaps that trigger vscode functionality via VSCodeNotify still work. So does moving the cursor with the mouse. 
If I open a different tab and go back everything works again.

This happens on any of the following:
`<c-h>`,`<c-j>`,`<c-k>`,`<c-l>`

**Comment by theol0403** [comment]
Do you have any custom nvim bindings for those? If so, they will break vscode-neovim.

**Comment by ghry5** [comment]
I double checked using `:nmap <c-j>`. Indeed I do have some mappings. I created a separate config just for vscode-neovim (to not mess with my default neovim configuration) under `.config/vscode-neovim/init.lua`. Turns out that even if I pass a custom config path to the `vscode-neovim.neovimInitVimPaths` option, it still picks up on my default neovim config path and loads all files in the `after/plugins` directory. 
This has some keybindings for navigating between tmux and neovim for me which causes my issue.

Since it is very easy to miss that neovim still loads the default `after/plugins` files even when pointing to a completely different `init.lua` i think there should be an option to reconfigure nvims `stdpath('config')` or at least have a section about `after/plugin` in the README.

For now I can edit my `after/plugin` files to early exit when `exists('g:vscode')` and not load those keybindings to fix my issue. Thank you alot for bringing me on the right track.
